app.py

```diff
@@ -150,7 +152,8 @@ def users_show(user_id):
                 .order_by(Message.timestamp.desc())
                 .limit(100)
                 .all())
-    return render_template('users/show.html', user=user, messages=messages)
+    likes = [message.id for message in user.likes]

```

Comment:
```suggestion
    liked_messages = [message.id for message in user.liked_messages]

```
The meta property `likes` on `User` should be called `liked_messages` since there's a model called `Likes`.